invokeai/backend/model_manager/cache.py

```diff
@@ -110,7 +98,7 @@ def __init__(
         sequential_offload: bool = False,
```

Comment:
`sequential_offload` appears to not be referenced

Reply:
fixed